presto-base-arrow-flight/src/test/java/com/facebook/plugin/arrow/TestArrowFlightQueries.java

```diff
@@ -83,11 +83,16 @@ public void close()
         allocator.close();
     }
 
+    public static int getServerPort()
+    {
+        return 9443;
```

Comment:
Use PortUtil

Reply:
Updated native to use PortUtil to get a free port